### ISSUE TITLE
Loader NUMBER_OF_THREADS

### ISSUE THREAD

**Issue by feamcor** [k2view-academy/K2View-Academy#1204]
On a Fabric installation where Cassandra is **NOT** being used as system DB and as MDB storage layer, what would be the impact of the `NUMBER_OF_THREADS` parameter within the `[default_loader]` section and/or any of the other custom loader sections (e.g., `parser_loader`, `iid_finder_loader`, `batch_process_loader`, etc.)?

Another way of asking would be if there is a loader for Postgres and SQLite (as possible system DBs) and for object storages like S3 and ABS (as possible MDB storage layers)?

Please advise.
Thanks.

**Comment by natalylic** [comment]
A new section will be added (in V8.2), planned to be release in Dec/2024, and it is applicable for JDBC system DB (PostgreSQL or SQLite):
```
[jdbc_default_loader]
#MODE=BATCH
#QUEUE_SIZE=10000
#BATCH_SIZE=1000
```

**Comment by feamcor** [comment]
So, what about the previous 7 and 8 versions that already accept Postgres and SQLite?
Is it fair to assume that `[default_loader]` has no impact on them?

**Comment by natalylic** [comment]
In these versions, the `[default_loader]` section has configuration that fits Cassandra.
So it is recommended to update the above parameters manually to get the best performance for other system DB types.

**Comment by feamcor** [comment]
Hi @natalylic, but to update where?
I feel that you are missing my question... until the `jdbc_default_loader` is available on v8.2, does any of the existing (e.g., `default_loader`, `parser_loader`, `iid_finder_loader`, `batch_process_loader`) or custom loader sections actually have any impact on an installation where Postgres or SQLite are used as system DB?
Please advise.
Thank.

**Comment by mironabr** [comment]
hi @feamcor ,
All the above sections actually inherited [default_loader].  The default values in that section are good for Cassandra as system_db.
Until 8.2 will be released, what Nataly wanted to say that you have to update by yourself the values in [default_loader] (or any other mentioned section) to gain the best performance when using sqlite or PG using the values that will be the default in 8.2:
[default_loader]
MODE=BATCH
#QUEUE_SIZE=10000
BATCH_SIZE=1000
